addons/cookoff/XEH_postInit.sqf

```diff
@@ -81,6 +81,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 ["AllVehicles", "killed", {
     params ["_vehicle"];
     if (_vehicle getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
+        if (GVAR(ammoCookoffDuration) == 0) exitWith {};
```

Comment:
Since that setting existed in 3.8, I would like to keep for now.
But when we switch to CBA settings, they could be merged.
I changed string table to indicate it.